Jenkinsfile

```diff
@@ -819,7 +819,8 @@ pipeline {
                                            if [ -z "$test_tag" ]; then
                                                test_tag=regression,vm
                                            fi
-                                           ./ftest.sh "$test_tag" ''' + env.NODELIST,
+                                           tnodes=$(echo $NODELIST | cut -d ',' -f 1-9)
```

Comment:
FWIW, you can slice up comma separated lists in bash without a fork/exec with the following:

```bash
NODELIST=(${NODELIST//,/ })  # arrays have a different namespace from variables
tnodes=${NODELIST[@]:0:9}
```